### ISSUE TITLE
Is there a way to get metasploit's webhook for a Discord server?

### ISSUE THREAD

**Issue by Elmo99** [rapid7/metasploit-framework#10333]
I recently created a Discord server that i still in progress, i'm tying to set up a webhook to metasploit-framework so that on channel #metasploit message is sent when something is updated. By far i have only done this by forking this by myself but i would like to webhook this exact repository. Is that possible?

By the way, anyone into pentesting, cracking should join my upcoming Discord server.
Heres a link: https://discord.gg/PEkXBqx
It requires a password when you first join the server, this is sort of anti-leech stuff.
The password is "Crackhat" in Base64

**Comment by wvu-r7** [comment]
I don't think this is a Metasploit bug? Normally we track issues here. Can you ask on IRC or send an e-mail?
